packages/email-components/.stylelintrc.js

```diff
@@ -4,6 +4,7 @@
 module.exports = {
   extends: ['@lmc-eu/stylelint-config', 'stylelint-prettier/recommended'],
   rules: {
+    'declaration-no-important': null,
```

Comment:
Since this is a js file, not only dummy json, therefore it is possible to add a comment why the rule is disabled, I'd place one here.